sys/include/evtimer.h

```diff
@@ -111,15 +122,23 @@ void evtimer_print(const evtimer_t *evtimer);
  */
 static inline uint32_t evtimer_now_msec(void)
 {
+#if IS_USED(MODULE_EVTIMER_ON_ZTIMER)
+    return ztimer_now(ZTIMER_MSEC);
+#else
     return xtimer_now_usec64() / US_PER_MS;
+#endif
 }
 
 /**
  * @brief   Return the current system time in minutes
  */
 static inline uint32_t evtimer_now_min(void)
 {
+#if IS_USED(MODULE_EVTIMER_ON_ZTIMER)
+    return ztimer_now(ZTIMER_MSEC) / (MS_PER_SEC * SEC_PER_MIN);
```

Comment:
this is a bit dangerous, on ztimer, this will overflow after 49 days. with xtimer_now_usec64(), not in our lifetimes.
A quickfix would be to depend on ztimer_now64.

Reply:
is there a user for this function? :)

Reply:
Good point! Done.
`gnrc_ipv6_nib` is a user of this method. ;)